client/src/pages/ServicesPage/ServicesPage.css

```
@@ -1,62 +1,51 @@
+.services-page {
+	margin: 0 20px;
+}
+
 .services-container {
 	display: flex;
 	flex-direction: column;
+  margin-bottom: 1rem;
 }
 
-.service-wrapper {
+.service-subcontainer {
 	display: flex;
-    flex-direction: column;
-    justify-content: space-between;
-    align-items: center;
-    gap: 1rem;
-    margin: 0 5%;
-}
-
-
-
-.service-title {
-	font-size: 1.2rem;
-	margin-left: 5%;
-    margin-top: 2rem;
+	flex-direction: column;
+	justify-content: space-between;
+  align-items: center;
+	gap: 1rem;
 }
 
-.title-line {
-	width: 90%;
-	height: 1px;
-	background-color: #333;
-	margin: 0 auto 10px auto;
-}
 
-.image-wrapper {
-  width: 350px;
-  height: 200px;
-  overflow: hidden;
+.image-container {
+	width: 350px;
+	height: 200px;
+	overflow: hidden;
 }
 
 .service-image {
-  width: 100%;
-  height: 100%;
-  object-fit: cover;
+	width: 100%;
+	height: 100%;
+	object-fit: cover;
 }
 
-.service-description{
-    text-align: justify;
-    width: 75%;
+.service-description {
+	text-align: justify;
+	width: 75%;
```

```suggestion
	width: 100%;
```
We can add for mobile version width: 100% and for desktop version: width: 75%;